scripts/test-app-is-running.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [ -z "$APP_SERVICE_HOST" ]; then
+  echo "APP_SERVICE_HOST is not set, unable to continue."
+  exit 1
+fi
+
+APP_SERVICE_URL=$APP_SERVICE_HOST/api/FastPurgeUrls
+
+# Use the env var set from the previous task with the path to the function app
+echo "Going to make request to function app: '$APP_SERVICE_URL'. "
+
+RESPONSE=func-app-response.json
+
+curl -sS -o "$RESPONSE" -XPOST "$APP_SERVICE_URL?code=$FUNCTION_KEY" -H "Content-Type: application/json" -d "@./scripts/resources/test-payload.json"
```

Comment:
Would an info/diagnostics endpoint in the function app (containing this check) be a better solution to working out if it is configured correctly. This would be more of a "Tell Don't Ask" approach.

Reply:
I don't think so. Function apps should be as light weight as possible and adding another end point goes against that idea. Plus, adding another endpoint doesn't get away from testing this endpoint.